grasa_event_locator/views.py

```diff
@@ -504,7 +509,7 @@ def register(request):
 def resetpw(request):
         i = 0
         if request.method == 'POST':
-                if User.objects.filter(username=request.POST['emailAddr']).exists():
+                if UserAccount.objects.filter(username=request.POST['emailAddr']).exists():
```

Comment:
Just curious, why did these queries need to change from `User` -> `UserAccount`?

Reply:
Regarding UserAccount, someone changed the import statement to import User as UserAccount. As such there were instances of "User" and "UserAccount" in the code. Any case of the above was switched to UserAccount for consistency (and to get things working again).

Reply:
Oh huh, I didn't even notice. Nice catch!